code/src/App.js

```diff
@@ -1,9 +1,79 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import MessageList from 'components/MessageList';
+import NewThoughts from 'components/NewThoughts';
+import AnimatedPage from 'components/AnimatedPage';
+import Footer from 'components/Footer';
+import './index.css';
 
 export const App = () => {
+  const [newThoughts, setNewThoughts] = useState('');
+  const [newLike, setNewLike] = useState(false)
+  const [messageList, setMessageList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    fetchMessages();
+  }, [newLike, loading]);
+
+  const fetchMessages = () => {
+    setLoading(false)
```

Comment:
Do you have somewhere where you set loading to "true"?  If not, do you need the loading/setLoading useState?